### ISSUE TITLE
Problem with http auth

### ISSUE THREAD

**Issue by vsychov** [Baldinof/roadrunner-bundle#2]
Hi,

thank you for this great works. I found some strange behavior, with enabled http_auth provider.
It's doesn't work with roadrunner, steps to reproduce:

1. create new symfony project `composer create-project symfony/website-skeleton test`
2. add baldinof/roadrunner-bundle
3. make file `config/packages/security.yaml`, that it looks like this:
```
security:
    encoders:
        Symfony\Component\Security\Core\User\User: plaintext

    providers:
        users_in_memory:
            memory:
                users:
                    user:  { password: 'test', roles: [ 'ROLE_ADMIN' ] }

    firewalls:
        main:
            anonymous: lazy
            provider: users_in_memory
            http_basic:
                realm: Secured Area

    access_control:
         - { path: ^/, roles: ROLE_ADMIN }

```

4. try open `/` route, with login `user` and password `test`, and authorization not work

**Comment by Baldinof** [comment]
Hi, thank you for opening an issue!

I will have a look today :)

**Comment by Baldinof** [comment]
I released v1.0.2 that should fix this. Can you check if it works for you?

**Comment by vsychov** [comment]
Thank you, works great! 👍
